### ISSUE TITLE
Mapping `waterloop`, `bekken`, `owl`, `waterlichaamcategorietype`, `map`

### ISSUE THREAD

**Issue by damianooldoni** [riparias/vmm-macrophytes-occurrences#15]
I refer to the following columns of sheet `meetplaatsen`:

NL original field | EN translation
--- | ---
`waterloop` | `waterbody`
`bekken` | `basin`
`owl` | `waterbodySurface`
`waterlichaamcategorietype` | `waterbodyCategory`

I grouped these water body categorization values in `dynamicProperties` within `events.csv`. These fields are not strictly speaking measurements or facts, so I preferred to group them in the JSON formatted field `dynamicProperties`.

Based on discussion with @peterdesmet, I also decided to not include `kaart` values as the reference to a map seems to me for internal use only.

**Translation and decision to be reviewed by VMM**.

**Comment by peterdesmet** [comment]
What do the codes stand for:

```
Bg	766
BgK	548
Bk	904
BkK	863
Bs	2
NVT	10
Pb	307
Pz	105
Rg	343
Rk	144
RtNt	79
Rzg	10
```

**Comment by damianooldoni** [comment]
The codes on the left are water category types as defined by VMM (see below) and used in all Flanders by other institutes as well. The right ones (numbers) I cannot find where you found them.

Waterbody category | meaning (NL)
--- | ---
Rzg | zeer grote rivier
Pz | zoete polderwaterloop
r | rivier 
BgK |  grote beek Kempen
BkK | Kleine Beek kempen
RtNt | probably linked to watercourses underlying a big change (waterlichaam = `NA`)
Rk - kleine rivier
Pb | brakke polderwaterloop
Bk | kleine beek
Bg | grote beek
Rg | grote rivier
NVT | niet van toepassing (waterlichaam = `NA`)
Bs | sterk brak meer

**Comment by peterdesmet** [comment]
The numbers are how often they occur, so don't worry about that. If those are codes as used by others, will leave them as is. A public reference for the codes could be useful though.

**Comment by damianooldoni** [comment]
Yes, I would leave the codes as well. The problem is that I couldn't find a unique reference for these categories online. I got (most of) them by searching on Google one by one: e.g. searching "waterlichaam type rzg VMM" brings me to a publid VMM document (PDF) containing the evaluation of a waterbody: https://www.vmm.be/bestanden/geoloketdata/SGBP/Beoordeling2/VL11_203_Beoordeel2.pdf

The type is described on top:
![image](https://user-images.githubusercontent.com/33662631/163470551-52ca384d-3abf-447b-bcd3-8c46d77a7758.png)

I will ask to Stijn Van Onsem, the LIFE RIPARIAS contact person at VMM, if he can find a better complete reference to these codes.

**Comment by damianooldoni** [comment]
Answer from Elina Bennetsen (VMM) via Stijn Van Onsem (VMM):

"""
Hier te vinden voor de 2e generatie stroomgebiedbeheerplannen : in de bestanden Vlaams deel van het internationaal stroomgebiedsdistrict Schelde en Vlaams deel van het internationaal stroomgebieddistrict Maas, beide onder 2. Analyses en beschermde gebieden (https://www.integraalwaterbeleid.be/nl/stroomgebiedbeheerplannen/stroomgebiedbeheerplannen-2016-2021/stroomgebiedbeheerplannen-voor-schelde-en-maas-2016-2021

 

Hier te vinden voor de 3e generatie stroomgebiedbeheerplannen (nog niet definitief goedgekeurd, maar is de meest up-to-date versie van de typologie): onder 2 analyses en beschermde gebieden -zowel hoofdtekst als bijlagen zullen info bevatten: https://www.volvanwater.be/documenten/beheerplan-vlaams-deel

 

Verder misschien interessant om het Achtergronddocument ‘beoordelingsmethoden’ bij de stroomgebiedbeheerplannen te bekijken als de macrofytendata worden gebruikt. Daarin staat beschrijving van meetstrategie plus uitleg over hoe per type gescoord wordt. Die info wordt per plancyclus up-to-date gebracht:

Versie van 2e sgbp: https://www.integraalwaterbeleid.be/nl/stroomgebiedbeheerplannen/stroomgebiedbeheerplannen-2016-2021/documenten/achtergronddocumenten/achtergronddocument-beoordeling-ecologische-en-chemische-toestand

Versie van 3e sgbp, zie pdf in bijlage (de achtergronddocumenten staan voorlopig niet meer online tot de finale goedkeuring van de plannen).
"""
